app/mailers/provider_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class ProviderMailer < ApplicationMailer
+  def request_created_email
+    @provider = params[:linked_token].entity.provider
+    @request_url = 'localhost:3000/' + 'request/' + params[:linked_token].entity.id.to_s
```

Comment:
localhost:3000 won't work in the final system right. I think we need [something like this](https://stackoverflow.com/questions/798917/how-do-i-configure-the-hostname-for-rails-actionmailer)?

Reply:
I _think_ this comment might be misplaced but if not, this is the URL that will route to viewing a request to be shared on social.

As for actionmailer config, I think I only edited the development config. I can do production config in the sendgrid setup work.